File: r8ted/Models/Review.cs
namespace r8ted.Models
{
    public class Review
    {
        public int Id { get; set; }
        public string User_id {get; set;}
        public int Movie_id { get; set; }
        public string Description { get; set;}
        public string Title { get; set;}
        public int Likes {get; set;}
        public bool IsPrivate { get; set;}
        public bool JoshRated { get; set;}

        public Profile Creator { get; set;}
    }
}

File: r8ted.client/src/models/Review.js
export class Review {
    constructor(data) {
        this.id = data.id
        this.userId = data.user_id
        // NOTE ADD MORE CREATOR INFO IF NEEDED HERE TO GET INFO PER REVIEW CARD
        this.userImg = data.creator.picture
        this.userName = data.creator.name
        this.movieId = data.movie_id
        this.title = data.title
        this.description = data.description
        this.likes = data.likes
        this.catOne = data.catOne
        this.catTwo = data.catTwo
        this.isPrivate = data.isPrivate
        this.joshRated = data.joshRated

    }
}